legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/relationalMappingExecution.pure

```diff
@@ -81,11 +81,13 @@ function meta::relational::mapping::generateInstantiationExecutionNode(sq:meta::
                                                                   offset = $paths->indexOf($p),
                                                                   sourceDataType = if($p.second.relationalType->isNotEmpty(),
                                                                                       | $p.second.relationalType,
-                                                                                      | $p.second.propertyMapping->match([
+                                                                                      | let sourceDataTypeFromMapping = $p.second.propertyMapping->match([
```

Comment:
As discussed, can we handling this by changing the match statement to handle more precise multiplicity cases